controllers/productController.js

```diff
@@ -48,7 +49,8 @@ const deleteProduct = async (req, res) => {
   res.status(StatusCodes.OK).json({ msg: 'Success! Product removed!' });
 };
 const uploadImage = async (req, res) => {
-  // console.log(req.files);
+  console.log(req.files);
+  // AKOSREVIEW: We should do a check for req.files.image existing too
```

Comment:
I notice that if I use Postman to send a file that is not named `image`; then this endpoint doesn't correctly find that (I would expect it to return "No file uploaded"). So I think that's because we're not checking for `!req.files.image` as well on line 54.

Reply:
@akosasante just to clarify: when mentioning the "image," you are referring to a key or something else? I've tried in postamn to send another key- 'foto' and it returned  "No file uploaded"
![image](https://github.com/NatachaKey/ecommerce/assets/103755090/7cdb6606-ac01-4a54-a8d6-9464bd0c5ac9)
after adding the check for   !req.files.image the output is the same
![image](https://github.com/NatachaKey/ecommerce/assets/103755090/11f6d9a4-4ad9-4cd9-9192-e6e2d6b2bf15)

Reply:
@akosasante sorry I 've messed up with review and comments option on github- just clicked 'finish review' to continue adding normat comments😅

Reply:
oh interesting; this is what I get, if I try to use a different key :thinking:  I'm not sure what I'm doing differently. It looksl ike in your code in the screenshot you are at least checking for `!req.files.image` not existing on line 57. So it's probably fine as is :+1: 

![](https://github.com/NatachaKey/ecommerce/assets/9094098/83582ac0-49f4-46a9-b914-749cd901dbee)